app/controllers/user_controller.py

```diff
@@ -126,4 +127,35 @@ def delete():
 
 @jwt_required()
 def patch():
-    ...
+    authorized_keys = ["email", "phone", "name", "password"]
```

Comment:
I believe we can add a "confirm password" option when the "password" key is present in the update, but i also think this should be done later as an extra.

Reply:
I've created a task for this on [Trello](https://trello.com/c/rEej1eLI).